src/olympia/ratings/tests/test_views.py

```diff
@@ -848,15 +848,19 @@ def test_list_addon_queries(self):
             with mock.patch('olympia.ratings.views.RatingViewSet'
                             '.get_addon_object') as get_addon_object:
                 get_addon_object.return_value = self.addon
-                response = self.client.get(self.url, {'addon': self.addon.pk})
+                response = self.client.get(
+                    self.url, {'addon': self.addon.pk, 'lang': 'en-US'})
```

Comment:
does the `lang` param make a difference to the response?

Reply:
Now that we're returning the `name` of the `addon` it does, we hit https://github.com/mozilla/addons-server/issues/8831 otherwise and generate more queries.